packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.test.tsx

```diff
@@ -265,18 +258,15 @@ describe('CompleteResetPassword page', () => {
         mockCompleteResetPasswordParams.email
       );
 
-      // TODO: y u no pass?
-      // expect(mockNavigate).toHaveBeenCalledWith(
-      //   `/account_recovery_confirm_key${getSearch({
-      //     mockToken,
-      //     mockCode,
-      //     mockEmail,
-      //     mockPasswordHash,
-      //   })}`,
-      //   {
-      //     replace: true,
-      //   }
-      // );
+      await waitFor(() => {
+        expect(mockNavigate).toHaveBeenCalledWith(
+          expect.stringContaining('/account_recovery_confirm_key'),
```

Comment:
We're currently doing different things regarding parameters in our tests for reset PW at the moment, and it makes it challenging to be consistent. If I mock `window.location.search` here to more explicitly test what this is being called with, the expired or damaged link is rendered instead due to the test set up. However as-is, `window.location` renders as `undefined` so to get this passing I'd have to test `.toHaveBeenCalledWith('/account_recovery_confirm_keyundefined'` or something like that. It seemed fine to test `stringContaining` for now (and `toContain` further down) and revisit our tests later when we do any architecture-related refactoring.